Makefile

```diff
@@ -33,12 +33,16 @@ VERSION ?= master
 
 TAG_LATEST ?= false
 
+# The version of restic binary to be downloaded for power architecture
+RESTIC_VERSION ?= 0.9.5
+
 ###
 ### These variables should not need tweaking.
 ###
 
 CLI_PLATFORMS := linux-amd64 linux-arm linux-arm64 darwin-amd64 windows-amd64 linux-ppc64le
```

Comment:
let's move L43-45 up to starting at L39 (above the "should not need tweaking" comment), and change them all to `?=` instead of `:=`. This is specifically necessary for `MANIFEST_PLATFORMS`, because when building a container locally for development purposes I want to override the value of this variable so I don't have to build a Power image just to push my amd64 dev image.

FYI, this is the command I'm running for dev purposes, that should allow me to build&push *just* an amd64 image:

```
REGISTRY=steveheptio VERSION=foo MANIFEST_PLATFORMS=amd64 make container push manifest
```